is.js

```diff
@@ -265,6 +265,30 @@
         if (is.all.boolean(value, other)) {
             return value === other;
         }
+        // check object
+        if (is.all.object(value, other)) {
+            var keys = Object.keys(value),
+                length = keys.length,
+                key;
+
+            if (is.empty(value) && is.empty(other)) {
+                return true;
+            }
+
+            if (length !== Object.keys(other)) {
```

Comment:
This is wrong. Maybe you wanted to write ```length !== Object.keys(other).length```

Reply:
I agree, in the next commit https://github.com/arasatasaygin/is.js/pull/262/commits/382f695bd5f10a04b1bbe0d7a2dcaf82f4fa3e08  I changed it.